src/components/EndangeredSpecies/FootNotes.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  habitat: 'Water, wrackline, lowbeach, midbeach, highbeach, foredune, and dune',
+  fledge: 'Juveniles that can fly but are still with parent.',
+  streets: 'Include permanent lifeguard towers, nearby streets to describe location',
+  banding:
+    "Banding: Left to Right, Top to Bottom (from bird's perspective). Ex: yellow, tan on " +
+    'left leg and green and green on right leg = yn:gg (all below ankle) or black above ' +
+    'ankle, pink on left leg and blue, yellow on right = Kp:by. All bands above ankle ' +
+    'UPPERCASE and all below lowercase. If no band on one leg = X. Split band (two ' +
+    'or more color per band) = yellow, blue on left and blue/green split and orange on ' +
+    'right = yb:(b/g)o. Color Code: Aqua:a, Blue:b, Green:g, Black:k, ' +
+    'Brown/Tan/Buckskin:n, Lime:l, Orange:o, Pink:p, Red:r, Silver:s, Violet:v, ' +
+    'White:w, Yellow:y. Only use x if one leg is unbanded and the other is banded. ' +
+    'Sex: Male, Female, Unknown',
+  nest: 'C/N = copulation/mating and nest construction, I = incubation, H = hatchling, B = brooding, O = other (specify in notes)',
+  behavior:
+    'R = roosting, F = foraging, S = sitting as if on nest, C=courting (scraping, ' +
+    'dancing, & fish presentation, shell tossing), Fishing = F, Flying = FL, Flushing = ' +
+    'FLU, Distraction Behavior = DB, Feeding = FE, O = other (specify in notes)',
```

Comment:
Based off the figma, the dropdown menu options were displayed in full, so should we keep the abbreviations for the tootip?